File: Easy/Java/2932. Maximum Strong Pair XOR I.java
class Solution {
    public int maximumStrongPairXor(int[] nums) {
        int xor = 0;
        Arrays.sort(nums);

        for(int x : nums){
            for(int y : nums){
                xor = Math.abs(x - y) <= Math.min(x, y) ? Math.max(xor, x ^ y) : xor;
            }
        }

        return xor;
    }
}

File: Easy/Python/2932. Maximum Strong Pair XOR I.py
class Solution(object):
    def maximumStrongPairXor(self, nums):
        xor = 0
        nums.sort()

        for x in nums:
            for y in nums:
                xor = max(xor, x ^ y) if abs(x - y) <= min(x, y) else xor
        
        return xor
